llvm/lib/Target/AArch64/AArch64.td

```diff
@@ -1420,7 +1420,8 @@ def ProcessorFeatures {
                                  FeaturePerfMon, FeatureSPE, FeatureSPE_EEF];
   list<SubtargetFeature> R82  = [HasV8_0rOps, FeaturePerfMon, FeatureFullFP16,
                                  FeatureFP16FML, FeatureSSBS, FeaturePredRes,
-                                 FeatureSB];
+                                 FeatureSB, FeatureRDM, FeatureDotProd,
+                                 FeatureComplxNum, FeatureJS];
```

Comment:
The addition of `FeatureComplxNum` and `FeatureJS` here doesn't seem to have any relation to the commit message. I can see that you've removed `FeatureRDM` and `FeatureDotProd` from `HasV8_0rOps` and instead put them in `R82`, but there's no corresponding removal of these two anywhere else to match the addition of them here.

If that was intentional, can you add something to the commit message explaining why it was needed?

Reply:
These were removed from `HasV8_0rOps` (in the v8.3 section), but github's highlighting of that change looks like it's been confused by the re-flow to 80 columns.

Reply:
Oh yes, now I see it! Thanks.